portality/templates/doaj/index.html

```diff
@@ -45,26 +45,34 @@ <h1>Find open access journals & articles.</h1>
             <section class="row mobile-scrollable-row">
                 <h2 class="sr-only">DOAJ in numbers</h2>
                 <p class="col-md-2 stat">
-
-                        <span class="stat__number">{{ statistics.countries }}</span>
-                        <span class="stat__description">countries represented</span>
-
+                    <span class="stat__number">{{ statistics.countries }}</span>
+                    <span class="stat__description">countries represented</span>
                 </p>
                 <p class="col-md-2 stat">
+                    {# link to query for journals published in 2019, need to find a better way to do this #}
```

Comment:
```suggestion
                    {# TODO: link to query for journals published in 2019, need to find a better way to do this #}
```
We could make this a redirect with a pre-defined query like those below, with the `gte` threshold value coming from config. Todo added so it gets flagged in IDE.